inst/help/raterAgreement.md

```diff
@@ -1,28 +1,32 @@
-Cohen's and Fleiss' kappa
+Rater Agreement
 ==========================
 
-Cohen's and Fleiss' kappa are coefficients that measure the agreement between raters on a nominal or ordinal scale (Cohen, 1960; Cohen, 1968; Fleiss, 1971). Cohen's kappa is limited to measure the agreement between two raters. Fleiss' kappa measures the agreement between two or more raters.
+Cohen's and Fleiss' kappa and Krippendorff's alpha are coefficients that measure the agreement between raters on a nominal or ordinal scale (Cohen, 1960; Cohen, 1968; Krippendorff, 1970; Fleiss, 1971).
+ Cohen's kappa is limited to measure the agreement between two raters. Fleiss' kappa and Krippendorff's alpha measure the agreement between two or more raters.
 
 *Rater* can refer to different judges, tests or other forms of rating here and in the section below.
 
 ### Input
 -------
 
-- Variables: The variables / columns to compute kappa for. Each variable corresponds to one rater, with different rows corresponding to different subjects being rated. 
-The maximum number of raters for Cohen's kappa is 2. If more than 2 variables/raters are entered, all possible pairs of the variables/raters will be calculated.
+- Variables: The variables / columns to compute kappa for. Each variable corresponds to one rater, with different rows corresponding to different subjects being rated. Note that for Krippendorff's alpha this is different from
+the common format, which has one rater per row and one column of measurements per subject. For Cohen's kappa the maximum number of raters  is 2. If more than 2 variables/raters are entered, all possible pairs of the variables/raters will be calculated. 
 
 - Cohen's kappa: Whether Cohen's kappa is calculated.
   - Unweighted or weighted: The unweighted kappa treats all disagreements equally, whereas the weighted kappa takes degrees of disagreement into account (Cohen, 1968). To calculate degrees of disagreement, a meaningful order of the rating categories is necessary. Therefore, weighted kappa can only be calculated for ordinal ratings. 
   
 - Fleiss' kappa: Whether Fleiss' kappa is calculated.
 
-- Confidence Interval: Whether a confidence interval should be reported for Cohen's and Fleiss' kappa and the size of the interval.
+- Krippendorff's alpha: Whether Krippendorff's alpha is calculated.
+  - Method: Krippendorff's alpha is calculated differently depending on the level of measurement of the variable. This option allows to select the method corresponding to the correct level of measurement.
```

Comment:
variable -> variables

Reply:
Well, JASP has only 3 types, but Krippendorff's alpha has 4?
![image](https://user-images.githubusercontent.com/61797391/165648371-642c39dc-c988-456f-bed6-dc6bde3dc718.png)

Reply:
I suppose even though the jasp variable type is always "scale" one can still assume it is interval or ratio?